ambari-server/src/main/python/ambari-server.py

```diff
@@ -571,6 +571,14 @@ def init_ldap_setup_parser_options(parser):
   parser.add_option('--ambari-admin-username', default=None, help="Ambari Admin username for LDAP setup", dest="ambari_admin_username")
   parser.add_option('--ambari-admin-password', default=None, help="Ambari Admin password for LDAP setup", dest="ambari_admin_password")
 
+@OsFamilyFuncImpl(OsFamilyImpl.DEFAULT)
+def init_setup_sso_options(parser):
+  parser.add_option('--sso-enabled', default=None, help="Indicates whether to enable/disable SSO", dest="sso_enabled")
+  parser.add_option('--sso-provider-url', default=None, help="The URL of SSO provider; this must be provided when --sso-enabled is set to 'true'", dest="sso_provider_url")
```

Comment:
as I remember, argparse automatically uses the name with underscores, so "dest=" argument should be excessive

Reply:
Thank Dmitriusan! It does not make any harm if we have it (in fact it protects us if argparse's behavior changes).